src/runtime/composables/local/useAuth.ts

```diff
@@ -32,9 +32,12 @@ const signIn: SignInFunc<Credentials, any> = async (credentials, signInOptions,
   const { rawToken } = useAuthState()
   rawToken.value = extractedToken
 
-  await nextTick(getSession)
+  const { callbackUrl, redirect = true, external, withSession = true } = signInOptions ?? {}
+
+  if (withSession) {
```

Comment:
Could you please add type support for the new `withSession` option? Please also consider, that you will need to account for the fact that this value is only available for the `local` and `refresh` providers and should therefore not be added to the signIn function of the `authjs` provider.

Reply:
Hi Zoey. Sorry it took a while. In my mind I responded somewhere. I can add typing.

But can you direct / help me to how I do this "account for the fact that this value is only available for the local and refresh providers and should therefore not be added to the signIn function of the authjs provider"? Because I have no idea what you mean here.

Reply:
So where do I add the typing support?

Reply:
And how do I make sure this is not added to the signIn function of the authjs provider?

Reply:
Is this enough? https://github.com/sidebase/nuxt-auth/pull/702/files#diff-23ace85c1398da87fad352ed08fd7026dab0a96402ccc8b6c3e23b478c993f43

Reply:
@zoey-kaiser ?